### ISSUE TITLE
Reduce artifacts size generated by dashboard

### ISSUE THREAD

**Issue by adpaco-aws** [model-checking/rmc#483]
Currently generates about ~60MB of artifacts. We need to reduce artifacts size so that it can be put online. Depends on #482 .

**Comment by adpaco-aws** [comment]
In particular, there are two aspects we are worried about: Size and stability. As mentioned, files generated by litani runs is about ~60MB of space. Not all of these files are needed for an HTML webpage. However, two litani runs performed on the same data will differ in size. Tomorrow I will find out why and update this issue.

**Comment by adpaco-aws** [comment]
First, we only need the HTML folder from litani runs. At present, that is ~65MB.

The HTML content consists of four components:
 * `artifacts` (~24MB - folder): Not useful at the moment. Following links to artifacts will make you go further down in the book tree and show nothing in the end. This usually keeps `cbmc-viewer` reports in C projects.
 * `index.html` (~2MB - file): Essential.
 * `pipelines`(~28MB - folder): Shows commands dependency graphs for launched jobs. Not essential but nice to have for now. 
 * `run.json` (~11MB - file): Basically a log file for the litani run.

We now feel like we have a good understanding of artifacts size in litani. Files within a litani run are not big (e.g., SVG files for the dependency graph are 4KB) but one must take into account that we are testing 600+ examples in these runs in 1-3 stages.

**Comment by adpaco-aws** [comment]
For now we are going to remove the `artifacts` folder and the `run.json` file. The latter could help in debugging but this is not something we are going to do in CI for now.

**Comment by adpaco-aws** [comment]
#505 includes the removal of this content before publishing the documentation.
